main.py

```diff
@@ -0,0 +1,63 @@
+# Imports
+from flask import Flask, render_template
+import requests, json
+import pandas as pd
+import datetime
+import mysql.connector
+from sqlalchemy import create_engine
+import threading
+from data import select_user
+
+app = Flask(__name__)
+
+def main():
+	
+	# Calls the main() function every 300 seconds 
+	threading.Timer(300.0, main).start()
+
+	# Reading API data into Pandas DataFrame
+	df = pd.read_json(r"https://api.tfl.gov.uk/BikePoint")
+
+	# Removing unwanted columns from DataFrame
+	df = df.drop(["$type","url","placeType","children","childrenUrls"], axis=1)
+
+	# Extracting NbBikes, NbDocks and NbEmpty from API
+	def get_elements(dataframe):
```

Comment:
Its generally not good design to define a function inside a function

Reply:
Since I'm calling the API calls every 5 seconds using threading, I figured I should put that in a function since I'm not that familiar with crontab.